src/main/java/edu/usf/cutr/pelias/SearchRequest.java

```diff
@@ -144,11 +144,11 @@ public Builder setApiEndpoint(String apiEndpoint) {
          * Builds the SearchRequest using the specified parameters
          * @return the SearchRequest using the specified parameters
          */
-        public SearchRequest build() {
+        public SearchRequest build() throws java.io.UnsupportedEncodingException {
             StringBuilder builder = new StringBuilder();
             builder.append(mApiEndPoint);
             builder.append("?text=");
-            builder.append(mText);
+            builder.append( URLEncoder.encode(mText, "UTF-8") );
```

Comment:
Thinking even more...  :)  -  There really isn't a good reason to throw the Exception back to the user of the library.   There isn't anything they can do to change it,  and it's not based on the provided input.   So, I think we should surround with try/catch block and in catch print the exception to System.err and just use the unencoded text.